sunpy/visualization/visualization.py

```diff
@@ -36,6 +36,17 @@ def fn_itoggle(*args, **kwargs):
 
 
 def axis_labels_from_ctype(ctype, unit):
+    """
+    A function to return axis labels for the given coordinate type and unit
+    using python dictionary.
+    
+    Parameters:
+    ctype: coordinates type
+    unit: required units
+    
+    Returns:
+    Axis_label [unit]
+    """
```

Comment:
Unfortunately this is the wrong style of documentation strings we use within SunPy. 

You should have a look over: http://docs.astropy.org/en/stable/development/docrules.html#sections 

But in summary the style is:

```
Parameters
----------
x : type
   Description of parameter `x`.

Returns
-------
int
    Description of anonymous integer return value.
```

Reply:
Okay, I will try to stick to the standards. Thank you for specifying.